### PR TITLE
Add US version of business insider

### PR DESCRIPTION
**Pull request by MaxDall**
This adds the US version of Business Insider. Since the site originates from the US, i also changed the existing DE version from `BusinessInsider` -> `BusinessInsiderDE`.

### CODE REVIEW

File: src/fundus/publishers/us/business_insider.py [hunk 1]
```diff
@@ -15,20 +15,13 @@
...
-            //div[contains(@class, 'article-body')] 
-            //p[
-                not(
-                    ancestor::*[@class='bi-bulletpoints'] or
-                    mark[@class='has-inline-color has-cyan-bluish-gray-color'] or 
-                    @class='has-text-align-right'
-                )
-            ]
+            //div[contains(@class, 'content-lock-content')] 
+            /p[text() and not(contains(@class, 'disclaimer'))]
```
Comment (addie9800) [inline src/fundus/publishers/us/business_insider.py:24]: The paragraph_selector does not seem to cover all cases: e.g. https://www.businessinsider.com/trump-civil-fraud-verdict-whats-in-judge-engorons-decision-2024-2 , https://www.businessinsider.com/couples-splurge-on-guest-experience-luxury-weddings-2024-2

File: src/fundus/publishers/us/business_insider.py [hunk 2]
```diff
@@ -15,20 +15,13 @@
 
 class BusinessInsiderParser(ParserProxy):
```
Comment (addie9800) [inline src/fundus/publishers/us/business_insider.py:16]: Some articles do not contain meta keywords, but rather meta news_keywords e.g. https://www.businessinsider.com/guides/health/fitness/best-running-headphones

**Review by addie9800 — CHANGES_REQUESTED** [review]
Thanks for making the switch and adding the US version as well.

File: src/fundus/publishers/us/business_insider.py (hunk 1 above)
Comment (MaxDall) [reply src/fundus/publishers/us/business_insider.py:24]: I removed `text()` from the selector.

File: src/fundus/publishers/us/business_insider.py (hunk 2 above)
Comment (MaxDall) [reply src/fundus/publishers/us/business_insider.py:16]: I've updated the topic parsing accordingly.

**Review by addie9800 — DISMISSED** [review]
Except for the weird behavior I shared with you on slack but what I guess is unrelated to this specific PR, this looks good.

File: src/fundus/publishers/us/business_insider.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        def authors(self) -> List[str]:
+            return generic_author_parsing(self.precomputed.ld.bf_search("author"))
+
+        @attribute
+        def publishing_date(self) -> Optional[datetime.datetime]:
+            return generic_date_parsing(self.precomputed.ld.bf_search("datePublished"))
+
+        @attribute
+        def title(self) -> Optional[str]:
+            return self.precomputed.ld.bf_search("headline")
```
[the remark this answers is not in the document]
Comment (MaxDall) [reply src/fundus/publishers/us/business_insider.py:47]: 👍

**Review by addie9800 — DISMISSED** [review]
👍

**Pull request merged**